packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -167,13 +170,24 @@ export default defineComponent({
     runningUploads: 0, // all uploads (not files!) that are in progress currently
     bytesTotal: 0,
     bytesUploaded: 0,
+    uploadSpeed: 0,
     filesInEstimation: {},
     timeStarted: null,
     remainingTime: undefined
   }),
   computed: {
     ...mapGetters(['configuration']),
 
+    uploadDetails() {
+      if (this.uploadSpeed === 0) {
+        return ''
+      }
+      const uploadedBytes = filesize(this.bytesUploaded)
```

Comment:
Please use our `formatFileSize` helper from `web-pkg`. For the use case of showing upload speed I'd say you should introduce an optional third argument to that helper, `precision?: number`, with which you should override the internally calculated precision to `2`.
Reason: a) unified file size formatting across ownCloud Web and b) it forces you to pass the users locale into the formatting.

Reply:
Do you mean override the precision to `1` (which "limiting the decimal to 1 instead of 2" from your comment above suggests) since from how I understand https://github.com/avoidwork/filesize.js/blob/d41818f78b0e2023eec98e5f9964aaeefaf91e8a/src/filesize.js#L178 the default of the wrapped lib is already 2? :)

Reply:
our `formatFileSize` helper has a default of `1` for values smaller than 1 mb, and a default of `0` for values greater than 1 mb. That's somehow our default when showing file sizes (was decided by PM 1 or 2 years ago). I meant overriding **that** default, not the one of the filesize lib ;-)

Reply:
Got it, thx for the swift reply!